src/language/pattern-types.md

```diff
@@ -216,16 +227,19 @@ patterns.
 Null-check patterns match first if the value is not null, and then match the inner
 pattern against that same value.
 
+Use the null-check pattern to silently treat `null` values as match failures
+without throwing.
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).
```suggestion
To treat `null` values as match failures
without throwing errors, use the null-check pattern.
```

**note:** The `silently treat` and `without throwing` are redundant. I removed one.